static/src/javascripts/bootstraps/commercial.js

```diff
@@ -10,7 +10,9 @@ import closeDisabledSlots from 'commercial/modules/close-disabled-slots';
 import prepareGoogletag from 'commercial/modules/dfp/prepare-googletag';
 import prepareSonobiTag from 'commercial/modules/dfp/prepare-sonobi-tag';
 import prepareSwitchTag from 'commercial/modules/dfp/prepare-switch-tag';
-import fillAdvertSlots from 'commercial/modules/dfp/fill-advert-slots';
+import {
+    init as fillAdvertSlots,
```

Comment:
Let's avoid aliased imports and rename the export to something more appropriate?